src/components/Link/Stories_shim.css

```
@@ -0,0 +1,3 @@
+ul li {
+  padding: 1rem;
+}
```

You should use class names, else these styles are going to mess with the styles in other pages. But they'll only mess with those styles if you go to this page first, and then other pages, which will make it extra confusing. Thus, please use class names.
```suggestion
ul li.li-storybook-padding {
  padding: 1rem;
}
```

---

alternate css (because `has` is new and shiny):
```suggestion
ul li:has(.neo-link) {
  padding: 1rem;
}
```